docs/components.md

```diff
@@ -103,14 +103,63 @@ To learn more about lifecycle methods, [see the lifecycle methods page](lifecycl
 
 ### Syntactic variants
 
+#### Closure components
+
+One of the easiest ways to manage state in a component is with a closure. A "closure component" is one that returns an object with a view function and optionally other lifecycle hooks. It has the ability to manage instance state within the body of the outer function.
```

Comment:
Should we maybe clarify that it returns an actual POJO component?

Reply:
It doesn't - we aren't cloning it. It returns a component *instance*, where `this` is set to the object itself. Object components aren't themselves instances - it's `Object.create(Comp)` that's the instance. Just thanks to prototype delegation, object components look similar to their instances.

Reply:
100% agree. Clarity is my entire point here.